packages/components/src/internal/components/editable/LookupCell.tsx

```diff
@@ -161,18 +158,12 @@ export const LookupCell: FC<LookupCellProps> = memo(props => {
         );
     }
 
-    // if the column is a lookup, we need to pass the containerPath to the QuerySelect
-    const containerPath_ =
-        containerPath ??
-        getValueFromRow(row?.toJS(), 'Folder')?.toString() ??
-        getValueFromRow(row?.toJS(), 'Container')?.toString();
```

Comment:
Those selenium tests are passing. There are several editable grids (e.g. the DataClassEditableGridForUpdate) that add the Folder to the EditorModel columnMap if App.isProductProjectsEnabled is true. This code isn't needed because we grab the Folder from the EditorModel in the `inputCellFactory` if we're updating data _and_ there is a container value on the row.